### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by karleitner**
It's a naive implementation, production images only, not usable for local development. The images come out to about 20 MB for the client and 120 MB for the server at the moment. Might be useful for demoing and prod deployments. The database service can be used for development, I added the command in the README. Let me know what you think.

### CODE REVIEW

File: client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:13-alpine
```
Comment (iredjee) [inline client/Dockerfile]: Let's use v12 as LTS

File: client/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+WORKDIR /usr/src/app
+COPY . /usr/src/app/
+RUN set -ex; \
+  yarn install --frozen-lockfile --production; \
+  yarn run build
+
+FROM nginx:alpine
+WORKDIR /usr/share/nginx/html
+COPY --from=0 /usr/src/app/build/ /usr/share/nginx/html
```
Comment (iredjee) [inline client/Dockerfile]: New line is missing

**Comment by Guillaume-N** [comment]
Why would we need docker?

**Comment by karleitner** [comment]
@iredjee Thanks for the valuable feedback!

@Guillaume-N I figured it would be easier to spin up a demo instance, especially for people who don't want to bother with installing Node/Yarn or setting up Postgres. Now that you mention it, I should focus more on the core features of the project and getting stuff done there instead. :flushed: I left the client Dockerfile only like @iredjee suggested, but feel free to close the PR if you don't think it makes sense.

**Comment by iredjee** [comment]
@Guillaume-N is right, we don't need docker for now. So @karleitner , I'm closing this PR, sorry

**Pull request closed without merging**